Probesys/bileto
# imp: misc

Changes proposed in this pull request: see commits

How to test the feature manually:

1. check the new logo appears on the different pages and as favicon

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
